### ISSUE TITLE
[BUG] No orderbook exists for taker market on cross exchange and arbitrage on paper trade mode

### ISSUE THREAD

**Issue by PtrckM** [CoinAlpha/hummingbot#2335]
**Describe the bug**
Running same exchange on arb or xemm will get this error `ValueError: No order book exists` when starting 

**Steps To Reproduce**
create strategy choose arb or xemm
choose 1st exchange: binance
2nd exchange: binance
choose 1st pair: YFII-BUSD
2nd pair: YFII-USDT

```
2020-09-15 21:54:41,998 - 13436 - hummingbot.core.clock - ERROR - Unexpected error running clock tick.
Traceback (most recent call last):
  File "hummingbot\core\clock.pyx", line 119, in hummingbot.core.clock.Clock.run_til
  File "hummingbot\strategy\arbitrage\arbitrage.pyx", line 192, in hummingbot.strategy.arbitrage.arbitrage.ArbitrageStrategy.c_tick
  File "hummingbot\strategy\arbitrage\arbitrage.pyx", line 317, in hummingbot.strategy.arbitrage.arbitrage.ArbitrageStrategy.c_process_market_pair
  File "hummingbot\strategy\arbitrage\arbitrage.pyx", line 254, in hummingbot.strategy.arbitrage.arbitrage.ArbitrageStrategy.c_calculate_arbitrage_top_order_profitability
  File "hummingbot\strategy\market_trading_pair_tuple.py", line 37, in get_price
  File "hummingbot\connector\exchange_base.pyx", line 203, in hummingbot.connector.exchange_base.ExchangeBase.get_price
  File "hummingbot\connector\exchange_base.pyx", line 100, in hummingbot.connector.exchange_base.ExchangeBase.c_get_price
  File "hummingbot\connector\exchange\paper_trade\paper_trade_market.pyx", line 899, in hummingbot.connector.exchange.paper_trade.paper_trade_market.PaperTradeMarket.c_get_order_book
ValueError: No order book exists for 'YFII-USDT'.
```

**Screenshots**
- config
![image](https://user-images.githubusercontent.com/54516858/93221470-6506e480-f7a0-11ea-8966-162e1265ef86.png)

![image](https://user-images.githubusercontent.com/54516858/93221627-92539280-f7a0-11ea-9cb8-0bb4153e24a1.png)

- test on live trade no error

![image](https://user-images.githubusercontent.com/54516858/93221723-ae573400-f7a0-11ea-9396-0eff8ab4ec6b.png)
![image](https://user-images.githubusercontent.com/54516858/93221743-b616d880-f7a0-11ea-8068-d7192916e192.png)

**Release version**
0.31.0

**Attachments**
[logs_bytav.log](https://github.com/CoinAlpha/hummingbot/files/5225467/logs_bytav.log)
[logs_conf_xemm_2.log](https://github.com/CoinAlpha/hummingbot/files/5225471/logs_conf_xemm_2.log)

**Comment by RC-13** [comment]
Test this issue and found out that it happens with other market pair as well if you use the same exchange as maker and taker.
![image](https://user-images.githubusercontent.com/57189990/93271808-b5a13080-f7e6-11ea-9351-78888bce71ef.png)

**Comment by PtrckM** [comment]
Fix on **dev-0.32.0**

![image](https://user-images.githubusercontent.com/54516858/94010301-550e8680-fdd8-11ea-8389-bdd26189581a.png)
